modules/ROOT/pages/mule-message-structure.adoc

```diff
@@ -4,68 +4,183 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: studio, components, elements, message, mule message, architecture
 
-// [NOTE]
-// This document examines the Mule message in the context of a *flow* rather than a *batch job*. Please see xref:3.8@mule-runtime::batch-processing.adoc[Batch Processing] for more information about how messages in a batch job are broken up and processed as *records*.
+A Mule message is the part of the Mule event that serves as a container for
```

Comment:
same as in the previous page

Reply:
These has a lot in common with the other page. Can one of them be removed in favor of the other?

Reply:
I will be deleting this page. Thought I'd already done a git rm on it. Oh well.